api-docs/api-reference/methods/users/post-add-user-v2.0.rst

```diff
@@ -119,27 +119,27 @@ This table shows the body parameters for the request:
 |                          |                         |specifies the user           |
 |                          |                         |account information.         |
 +--------------------------+-------------------------+-----------------------------+
-|user.**username**         |String *(Required)*      |The name to assign to        |
+|user.\ **username**       |String *(Required)*      |The name to assign to        |
 |                          |                         |the user. Specify a          |
 |                          |                         |value that meets the         |
 |                          |                         |following criteria:          |
 |                          |                         |                             |
-|                          |                         |* Start with an alpha \      |
-|                          |                         |  character.\                |
-|                          |                         |* Minimum length: 1 character|
-|                          |                         |* Can contain upper and      |
+|                          |                         |- Start with an alpha        |
+|                          |                         |  character.                 |
+|                          |                         |- Minimum length: 1 character|
+|                          |                         |- Can contain upper and      |
 |                          |                         |  lowercase characters.      |
-|                          |                         |* Can contain any of the     |
+|                          |                         |- Can contain any of the     |
 |                          |                         |  following special          |
-|                          |                         |  characters:. ``- @ _``     |
+|                          |                         |  characters: ``- @ _``      |
 |                          |                         |                             |
 +--------------------------+-------------------------+-----------------------------+
 |user.**email**            |String *(Required)*      |Email address for the        |
 |                          |                         |user account, for example    |
 |                          |                         |``jqsmith@test.com``         |
 |                          |                         |                             |
 +--------------------------+-------------------------+-----------------------------+
-|user.**enabled**          |Bool *(Required)*        |Indicates whether the        |
+|user.**enabled**          |Boolean *(Required)*     |Indicates whether the        |
```

Comment:
Are these missing the the '\'? Example user.\ **enabled**

Note: some other are also missing it.